entity-framework/core/modeling/inheritance.md

```diff
@@ -13,11 +13,11 @@ Inheritance in the EF model is used to control how inheritance in the entity cla
 
 By convention, it is up to the database provider to determine how inheritance will be represented in the database. See [Inheritance (Relational Database)](relational/inheritance.md) for how this is handled with a relational database provider.
 
-EF will only setup inheritance if two or more inherited types are explicitly included in the model. EF will not scan for base or derived types that were not otherwise included in the model. You can include types in the model by exposing a *DbSet<TEntity>* for each type in the inheritance hierarchy.
+EF will only setup inheritance if two or more inherited types are explicitly included in the model. EF will not scan for base or derived types that were not otherwise included in the model. You can include types in the model by exposing a `DbSet<TEntity>` for each type in the inheritance hierarchy.
 
 [!code-csharp[Main](../../../samples/core/Modeling/Conventions/InheritanceDbSets.cs?highlight=3-4&name=Model)]
 
-If you don't want to expose a *DbSet<TEntity>* for one or more entities in the hierarchy, you can use the Fluent API to ensure they are included in the model.
+If you don't want to expose a `DbSet<TEntity>` for one or more entities in the hierarchy, you can use the Fluent API to ensure they are included in the model.
```

Comment:
IIRC we can also escape it with backslash, I guess it's a separate question of how we refer to things. I'd go even farther and say that the generic <TEntity> adds nothing, we could just talk about DbSet in our documentation...

Reply:
Ah, it was for the generic. Yeah \*DbSet\\\<TEntity>\* would also work. I'm fine with backticks, escaping, or using the non-generic form. You decide

Reply:
I personally think we have too many jarring code fragments in our docs. I really preferred it if we just talked about contexts rather than `DbContext<T>` (or even `DbContext` or DbContext). For DbSet I'd go with the non-generic non-quoted version, and so forth.

But that seems like a decision to decide as a team.

Reply:
@bricelam @roji Was a decision made here?

Reply:
I guess it's a general docs question, not really lint-related - we're pretty inconsistent across our docs on this (the linter simply complained about the unescaped angle bracket). We can discuss in design.

Since this point is already pretty inconsistent across our docs, I can just change this back to bold (with backslash) and handle the general question in another issue.

Reply:
If you want to be consistent with what we do for dotnet/docs, APIs are either inside backticks (to prevent them from being localized) or they use the xref links to link to the API docs.